COPY/usr/bin/manageiq-messaging-ready

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+
+require "yaml"
+require "sd_notify"
+require "manageiq-messaging"
+require "manageiq-password"
+
+def service_ready?(host, port)
+  `ncat #{host} #{port} </dev/null 2>/dev/null`
+  $?.success?
+end
+
+def messaging_ready?(msg_yaml, env = "production")
+  host, port = msg_yaml[env].values_at("hostname", "port")
+  if service_ready?(host, port || 9092)
```

Comment:
Do we require hostname in the .yml file?  Does it have the same issues as the manageiq-db-ready when the .yml is missing some fields?  See: https://github.com/ManageIQ/manageiq-appliance/pull/367

Reply:
Note, we're not assigning `port` here so, if we default to 9092, we don't print the port 9092 in the `puts` below.

Reply:
@jrafanie Thanks, I've added default values for host and port since they would be required values